wp-autoupdates.php

```diff
@@ -731,15 +731,15 @@ function wp_autoupdates_debug_information( $info ) {
 				/* translators: %s: Latest plugin version number. */
 				$plugin_version_string       .= ' ' . sprintf( __( '(Latest version: %s)', 'wp-autoupdates' ), $plugin_updates[ $plugin_path ]->update->new_version );
 				/* translators: %s: Latest plugin version number. */
-				$plugin_version_string_debug .= ' ' . sprintf( __( '(latest version: %s)', 'wp-autoupdates' ), $plugin_updates[ $plugin_path ]->update->new_version );
+				$plugin_version_string_debug .= ' ' . sprintf( __( '(Latest version: %s)', 'wp-autoupdates' ), $plugin_updates[ $plugin_path ]->update->new_version );
 			}
 
 			if ( in_array( $plugin_path, $wp_auto_update_plugins ) ) {
 				$plugin_version_string       .= ' | ' . sprintf( __( 'Auto-updates enabled', 'wp-autoupdates' ) );
-				$plugin_version_string_debug .= sprintf( __( 'auto-updates enabled', 'wp-autoupdates' ) );
+				$plugin_version_string_debug .= sprintf( __( 'Auto-updates enabled', 'wp-autoupdates' ) );
```

Comment:
Small note related to the requested change in the other string: please do **not** change to lowercase here, your proposal is perfect :D

Reply:
@audrasjb Do I need to update patch or it's perfect?

Reply:
To clarify a bit: please update the patch to address the other issue (lowercase `Latest` to `latest`) but don't lowercase `Auto-updates` 😄

Reply:
Updated patch as requested

Reply:
Thanks! 👍